addons/stock/wizard/stock_request_count.py

```diff
@@ -9,18 +9,28 @@ class StockRequestCount(models.TransientModel):
     _description = 'Stock Request an Inventory Count'
 
     inventory_date = fields.Date(
-        'Inventory Date', required=True,
+        'Scheduled at', required=True,
         help="Choose a date to get the inventory at that date",
         default=fields.Datetime.now)
-    user_id = fields.Many2one('res.users', string="User", domain=lambda self: [('all_group_ids', 'in', self.env.ref('stock.group_stock_user').id)])
+    user_id = fields.Many2one('res.users', string="Assign to", domain=lambda self: [('all_group_ids', 'in', self.env.ref('stock.group_stock_user').id)])
     quant_ids = fields.Many2many('stock.quant')
-    set_count = fields.Selection([('empty', 'Leave Empty'), ('set', 'Set Current Value')], default='empty', string='Count')
+    show_expected_quantity = fields.Boolean(help='If the user can see the expected quantity or not', compute='_compute_show_expected_quantity', inverse='_set_show_expected_quantity')
+
+    def _compute_show_expected_quantity(self):
+        show_quantity_count = self.env['ir.config_parameter'].sudo().get_param('stock_barcode.show_quantity_count', default='False') == 'True'
+        for record in self:
+            record.show_expected_quantity = show_quantity_count
+
+    def _set_show_expected_quantity(self):
```

Comment:
It would be great to return directly here and do a clean override in stock_barcode